jpo-ode-svcs/src/main/resources/application.properties

```diff
@@ -1,61 +0,0 @@
-#General Properties
-#==================
-groupId=${project.groupId}
-artifactId=${project.artifactId}
-version=${project.version}
-#server.port = 8443
-#server.ssl.key-store = classpath:jpo_ode_keystore.jks
-#server.ssl.key-store-password = keystore_pass
-#server.ssl.key-password = key_pass
-
-spring.http.multipart.max-file-size=1MB
-spring.http.multipart.max-request-size=1MB

```

Comment:
note: these values were completely unused within the application. I did an exhaustive search with my Find All & my IDE's Find Usages to confirm this

Reply:
I checked some docs and github copilot and found the following:

> In a typical Spring Boot web application, the values are used to configure the handling of multipart file uploads. Here's how each value is used:
> 
> - `spring.http.multipart.max-file-size`: This property sets the maximum size allowed for uploaded files. If a file exceeds this size, the upload will be rejected.
> - `spring.http.multipart.max-request-size`: This property sets the maximum size allowed for multipart/form-data requests. This includes the total size of all files and form data in the request.
> 
> These properties are typically used by Spring's `MultipartResolver` to enforce file upload limits and ensure that the application does not accept files or requests that are too large.

I ran a test locally with Postman to try uploading a file via `http://localhost:8080/upload/bsmLogDuringEvent` to the [only endpoint](https://github.com/CDOT-CV/jpo-ode/blob/b7e62aecd6c769d342bfae38604c01e51e600ffa/jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/upload/FileUploadController.java#L89) with `MutlipartFile` or any `Mutlipart*`. While my request wasn't valid, the error message returned allowed me to confirm that we use the built-in `StandardServletMultipartResolver`, which would use the removed configuration properties.

Spring reads the configuration properties as long as an application.properties (or yaml) file is present in the application's working directory (see [External Configuration](https://docs.spring.io/spring-boot/docs/2.1.7.RELEASE/reference/html/boot-features-external-config.html#boot-features-external-config)). With this in mind, I think it's safer to migrate the properties in question to the new application.yaml so that we don't inadvertently open the file upload endpoint to some unknown attack vector or change existing expected behavior.  I've done this work in https://github.com/CDOT-CV/jpo-ode/pull/110/commits/e2447e2f62f016828d858df460e5cf2bd1a5c534.

However, it might be a good idea to log a work item to follow up on these configuration values and make sure we do our due diligence to figure out what the proper limits should be with multipart file resolution. That's something I can't do (yet), though.